SWIPs/swip-honey_to_money.md

```diff
@@ -42,10 +42,10 @@ In the current implementation, honey prices are hardcoded. When this price in co
 
 ## Rationale
 We need a way for prices in the Swarm to change over time. However, due to the nature of Swarm - specifically independent connections between pairs of nodes each with their individual accounting and payments - there is no clear price discovery mechanism. Furthermore, since Swarm nodes are expected to be conduits for both payments and data (buy from one peer and sell to the other) it is necessary for nodes to coordinate price changes with all their peers. In short, we need a mechanism for network-wide changes in pricing that the peer-to-peer routing and accounting does not provide.
-By externalising the price to an oracle, we can completely decouple the basic functioning of Swarm - which is heavily focused on data availability, delivery, and security considerations - from the price discovery. By building support for price oracles, we can coordinate price changes across the network, and set the stage for future experimentation with various mechanisms for setting prices, from DAOs, to bonding curves, to fixed prices. This SWIP makes no recommendation for the eventual pricing mechanism, it only prepares the network for adopting them in future.
+By externalising the price to an oracle, we can completely decouple Swarm's core functions from price discovery. By building support for price oracles, we can coordinate price changes across the network, and set the stage for future experimentation with various mechanisms for setting prices, from DAOs, to bonding curves, to fixed prices. This SWIP makes no recommendation for the eventual pricing mechanism, it only prepares the network for adopting them in future.
 
 ## Backwards Compatibility 
-This SWIP is backward compatible as long as the price oracle quotes the same prices as is currently hard-coded in non-upgraded Swarm nodes. It is up to the owner of the MsgToHoney oracle to ensure that he does not update prices too much while not all nodes that run are on the new Swarm version. Currently, the Swarm is not running with a live test net for settling prices, so we expect no problems with this SWIP if it is implemented before the Swarm will go live with price-incentivization. 
+This SWIP is backward compatible as long as the price oracle quotes the same prices as is currently hard-coded in non-upgraded Swarm nodes. It is up to the owner of the `MsgToHoney` oracle to ensure that prices are not updated too much while not all running nodes are on the new Swarm. Currently, Swarm is not running a Swap-enabled live test net, so we don't expect any problems with this SWIP if it is implemented before Swarm will go live with incentivization. 
```

Comment:
" It is up to the owner of the `MsgToHoney` oracle to ensure that prices are not updated too much while not all running nodes are on the new Swarm".

This sounds unclear to me. What means too much? Which *new Swarm*? I think this sentence should be rewritten (or explain to me what it means first).

Reply:
Is this even correct? Has this been copy-pasted? It says `MsgToHoney`, but this is the `MoneyToHoney` SWIP?

Reply:
This basically boils down to the very need of the price oracle, which is that prices may only be updated atomically or not at all. If half of the nodes run a Swarm version which does not include the changes as proposed by this SWIP, accounting differences will come into existence. Hence: in the situation where we would have a live SWAP-enabled Swarm, the honey/Money oracle provider should ensure not to change his price if not all nodes are listening to him yet. If this was not clear to you, I will rewrite it.

Reply:
> is this even correct?

No, it should indeed be honeyToMoney.

Reply:
I would even remove this sentence all together. This clearly is very relevant for the `msgToHoney` oracle, not so much here.